src/app/cube/collections/collections.component.scss

```diff
@@ -0,0 +1,123 @@
+
+@import '~_vars.scss';
+$sidebar-width: 350px;
+$content-width: calc(#{$max-width} - #{$sidebar-width});
+
+.hero {
+    position: relative;
+    overflow: hidden;
+    padding: 70px 10px 130px;
+    background: $blue-gradient;
+
+    &:after {
+        content: '';
+        background: white;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        opacity: 0.03;
+        mask: $cube-pattern repeat;
+    }
+
+    &:before {
+        content: '';
+        background: $wrapper-background;
+        position: absolute;
+        width: 100%;
+        height: 100vw;
+        top: 370px;
+        left: 0;
+        right: 0;
+        transform: skewY(-2deg);
+    }
+
+    .inner {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
+
+        .title__icon {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            text-align: center;
+            margin: auto;
+            margin-bottom: 20px;
+            background: white;
+            color: #4290ea;
+            width: 100px;
+            height: 100px;
+            border-radius: 50px;
+
+            svg {
+                width: 60px;
+                height: 60px;
+            }
+        }
+    }
+
+    // &:before {
```

Comment:
Can we delete this if we're not using it?

Reply:
Deleted.